packages/base/NumberInput/src/NumberInput/NumberInput.tsx

```diff
@@ -102,8 +93,13 @@ export const NumberInput = forwardRef<HTMLInputElement, Props>(
     return (
       <OutlinedInput
         classes={{
-          root: classes.root,
-          input: classes.input,
+          root: 'pr-0 cursor-text',
+          input:
+            // required to remove arrows in WebKit based browsers
+            `[&::-webkit-outer-spin-button]:appearance-none [&::-webkit-outer-spin-button]:m-0 
+            [&::-webkit-inner-spin-button]:appearance-none [&::-webkit-inner-spin-button]:m-0 ` +
```

Comment:
NIT: We can remove the backticks here, as there is no interpolation
```suggestion
            '[&::-webkit-outer-spin-button]:appearance-none [&::-webkit-outer-spin-button]:m-0 
            [&::-webkit-inner-spin-button]:appearance-none [&::-webkit-inner-spin-button]:m-0 ' +
```

Reply:
This is mostly a stylistic choice, both will work the same, but it is usually agreed in the community to only use backticks when you need an interpolation (`${variable}`) or breaking lines, otherwise, we should use single quotes (or double quotes, depending on the project, for Toptal projects we decided on single quotes). We should avoid concatenating strings with the `+` sign. There was a time when concatenating strings with plus sign was the preferred way, but with the advent of es2015, template literals became the norm.

For this case in specific, using the plus sign was justified because we had an explanation comment after the plus sign, this is not possible with template literals. The backticks are unnecessary however, because we don't have variable interpolations nor multiple lines. This is why I also marked the comment as [a `NIT`](https://stackoverflow.com/questions/27810522/what-does-nit-mean-in-hacker-speak)

Reply:
Thank you for the explanation 👍